pom.xml

```diff
@@ -125,6 +125,12 @@
             <version>1.3.5</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
This is almost certainly not the correct dependency. If this is for the `@Nullable` annotation, then you probably instead want:
```xml
            <dependency>
                <groupId>com.google.code.findbugs</groupId>
                <artifactId>jsr305</artifactId>
                <version>3.0.2</version>
            </dependency>
```